src/ol/geom/linestring.js

```diff
@@ -167,6 +167,21 @@ ol.geom.LineString.prototype.getCoordinates = function() {
 
 
 /**
+ * Return a point interpolated along the linestring, `fraction` is a number
+ * between 0 and 1 representing fraction of total length of linestring.
+ * @param {number} fraction Fraction.
+ * @param {ol.Coordinate=} opt_dest Destination.
```

Comment:
I think "destination" is a bit confusing.  How about "Optional coordinate whose values will be modified.  If not provided, a new coordinate will be returned."?